ksql-rest-app/src/include/docker/run

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+
+#
+# Copyright 2019 Confluent Inc.
+#
+# Licensed under the Confluent Community License (the "License"); you may not use
+# this file except in compliance with the License.  You may obtain a copy of the
+# License at
+#
+# http://www.confluent.io/confluent-community-license
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OF ANY KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations under the License.
+#
+
+
+echo "===> Configuring ksqlDB..."
+printenv
```

Comment:
Is this for debugging? Might not want to do this in case someone is putting something sensitive in an environment variable.

Reply:
yeah this is for debugging - I'm doing it because it looked like the other CP systems were:
```
schema-registry_1  | ===> ENV Variables ...
schema-registry_1  | ALLOW_UNSIGNED=false
schema-registry_1  | COMPONENT=schema-registry
schema-registry_1  | CONFLUENT_DEB_VERSION=1
schema-registry_1  | CONFLUENT_MAJOR_VERSION=5
schema-registry_1  | CONFLUENT_MINOR_VERSION=4
schema-registry_1  | CONFLUENT_MVN_LABEL=-beta1
schema-registry_1  | CONFLUENT_PATCH_VERSION=0
...

I'm happy to remove this. Thoughts?
schema-registry_1  | _=/usr/bin/env
```

Reply:
I've removed this to unblock the PR, we can always add it back later